test/common/router/config_impl_test.cc

```diff
@@ -2241,12 +2241,16 @@ name: foo
   - name: redirect
     domains: [redirect.lyft.com]
     routes:
-      - match: { prefix: /foo }
+      - match: { prefix: /host_path_https }
+        redirect: { host_redirect: new.lyft.com, path_redirect: /new_path, https_redirect: true }
+      - match: { prefix: /host_path }
```

Comment:
based on the previous test coverage on host and path redirect combined, then host and path independently, would it be beneficial to do all chose two combinations here? i.e add the missing host_https path_https redirects?